docs/variants/novacustom_ns5x/test-matrix.md

```diff
@@ -63,7 +63,7 @@ subjected from before the release of the new binary.
 |:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
 | 1.   | [TPM Support][TPM]                                | TPM           | All                                  |
 | 2.   | [Measured Boot support][MBO]                      | MBO           | All                                  |
-| 3.   | [Secure Boot support][SBO]                        | SBO           | All                                  |
+| 3.   | [Secure Boot support][SBO]                        | SBO           | SBO002.001, SBO002.002               |
```

Comment:
@sulewskiprzemyslaw I think it would be nice to include the `SBO001.001` for all platforms with SB enabled.

Reply:
Added